data/customer_mocks.rb

```diff
@@ -0,0 +1,29 @@
+require './data/mockable'
+
+class CustomerMocks
+  extend Mockable
+
+  def self.customers_as_mocks(eg, customer_hashes = customers_as_hashes)
+    mock_generator(eg, 'Customer', customer_hashes)
+  end
+
+  def self.customers_as_hashes(number_of_hashes: 10, random_dates: true,
+                               first_name: 'First Name',
+                               last_name: 'Last Name',
+                               created_at: created_at_proc,
+                               updated_at: updated_at_proc)
```

Comment:
Am wondering if there may be a neater way of formatting this? It looks great as it is! Just wondering if it would look better aligned underneath 'random_dates' instead.